data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2707,6 +2707,21 @@
     [/filter_second]
 #enddef
 
+    # Filter how many times Malifor got attacked
+    [event]
```

Comment:
It's a bit odd to put this event between the above macro and its use below. Would make more sense to put it right next to the related event further below.